recipe/meta.yaml

```diff
@@ -1,57 +1,43 @@
-{% set version = "2.9.0" %}
+{% set name = "time-machine" %}
+{% set version = "2.10.0" %}
 
 package:
-  name: time-machine
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  - folder: dist
-    url: https://pypi.io/packages/source/t/time-machine/time-machine-{{ version }}.tar.gz
-    sha256: 60222d43f6e93a926adc36ed37a54bc8e4d0d8d1c4d449096afcfe85086129c2
-  - folder: src
-    url: https://github.com/adamchainz/time-machine/archive/refs/tags/{{ version }}.tar.gz
-    sha256: 4c356560a2af1b3235da6deeeb68de67afda2bcfbe2a810e211ae4c5df3684a4
-
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/time_machine-{{ version }}.tar.gz
+  sha256: 64fd89678cf589fc5554c311417128b2782222dd65f703bf248ef41541761da0
 
 build:
-  skip: true  # [py<37]
-  script: cd dist && {{ PYTHON }} -m pip install . -vv
+  noarch: python
```

Comment:
as this still uses [extension modules](https://github.com/adamchainz/time-machine/blob/2.10.0/setup.py#L16) it cannot be noarch.

Reply:
Thanks @bollwyvl I see, Let me try again to see if I can get this build working - Will start over with a new PR